tails.rst

```diff
@@ -0,0 +1,22 @@
+Using the most current Electrum on Tails
+========================================
+
+Tails currently ships with an outdated and insecure version of Electrum that can no longer be used on the public Electrum server network as of March 2019. Unfortunately installed software on Tails cannot be permanently upgraded since it is a fixed (read-only) image - you have to wait for Tails to upgrade.
```

Comment:
I would disagree with 3.1.3 being insecure, let's not say that.
```suggestion
Tails currently ships with an outdated version of Electrum that can no longer be used on the public Electrum server network as of March 2019. Unfortunately installed software on Tails cannot be permanently upgraded since it is a fixed (read-only) image - you have to wait for Tails to upgrade.
```